substra/cli/printers.py

```diff
@@ -116,6 +116,16 @@ def get_value(self, item, expand=False):
         return ''
 
 
+class ComputePlanProgressField(Field):
```

Comment:
What about creating a `ProgressField` with the following constructor for instance?

```python
def __init__(self, name, progress_ref, total_ref):
```

It will be more generic and could be used in other asset?

Reply:
Sure, why not!

Reply:
Done, thanks!